src/list.cpp

```diff
@@ -53,6 +53,38 @@ bool IsCreateStatement(const std::string& sql_statement){
 
 // LOGICAL DATABASE DESIGN
 
+void CheckJSONValuedAttribute(Configuration& state,
```

Comment:
I wanted to come up with a name that supported both json and store but I cant think of anything other than "object"